HW04/Ivan_Papushoi/array_my_methods_spec.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+require_relative 'ivan_papushoi_hw04.rb'
+require 'rspec/autorun'
+
+RSpec.describe Array do
+  describe '#my_select' do
+    context 'with array and odd? method' do
+      let(:array) { [1, 2, 3, 4, 5, 6] }
+      let(:term) { ->(x) { x.odd? } }
+      let(:expectation) { [1, 3, 5] }
+
+      it { expect(array.my_select(&term)).to eq(expectation) }
+    end
```

Comment:
For your case, `subject` works perfectly 😉
```suggestion
  describe '#my_select' do
    subject { array.my_select(&term) }
    context 'with array and odd? method' do
      let(:array) { [1, 2, 3, 4, 5, 6] }
      let(:term) { ->(x) { x.odd? } }
      let(:expectation) { [1, 3, 5] }

      it { is_expected.to eq(expectation) }
    end
```

Reply:
Yes, it looks better now. Thank you, Vlad